arm-network/2017-06-01/swagger/applicationGateway.json

```diff
@@ -390,7 +390,7 @@
         "tags": [
           "ApplicationGateways"
         ],
-        "operationId": "ApplicationGateways_ListAvailableSslOptions",
+        "operationId": "ApplicationGateways_AvailableSslOptions",
```

Comment:
AvailableSslOptions is new operation supported for application gateway which added in my previous PR (4c5dd5ded1f4a8cc26f7be35ad639c3e7380644d). I am undoing the name change as I might risk breaking something now.